### DISCUSSION TITLE
what is the archisteamfarm host address?

### DISCUSSION THREAD

**Discussion by jaxisaway** [Luois45/claim-free-steam-packages#33] [Q&A] [answered]
I've set up archisteamfarm but I do not understand what the host address means or how I'm meant to find it. It appears when you launch the application for the first time.
![image](https://user-images.githubusercontent.com/93849152/165107973-d849db06-9428-4227-89e9-c163905ed082.png)

Can anyone help me?

**Comment by louisa-uno** [comment, marked as answer]
The ArchiSteamFarm host address is the address which can be found using following documentation: https://github.com/JustArchiNET/ArchiSteamFarm/wiki/IPC

**Comment by PatrickJnr** [comment]
By default it should be **http://localhost:1242**
